Source/Xamarin/Prism.Unity.Forms/PrismApplication.cs

```diff
@@ -17,6 +17,8 @@ namespace Prism.Unity
 {
     public abstract class PrismApplication : PrismApplicationBase
     {
+        const string _navigationServiceName = "UnityPageNavigationService";
```

Comment:
I was up in the air about this.  I would prefer not to encourage extending the navigation service because it's logic is quite complex.  This should hardly ever be needed.  So I want to hide the name and only if someone really wants to extend it will they need to read the source code, or docs to extend it.